### DISCUSSION TITLE
Homebrew Formulae for podman not updated to v4?

### DISCUSSION THREAD

**Discussion by tedt10e** [containers/podman#13355] [Q&A] [answered]
`brew install podman` is still installing v 3.4.4.

https://formulae.brew.sh/formula/podman

Is there anything I can do to install v4 using homebrew?

Thanks

**Comment by tedt10e** [comment, marked as answer]
I found this conversation on homebrew. I believe I'll have to wait.

https://github.com/Homebrew/homebrew-core/pull/95318

**Comment by rhatdan** [comment]
Yes a couple more days hopefully.

**Comment by afbjorklund** [comment]
If you are looking for a version of "qemu" with support for virtfs, a formula can be found here:

https://github.com/afbjorklund/homebrew-core/blob/qemu-9p-darwin/Formula/qemu.rb

By default it will use the regular v6.2.0, but it can also build `--HEAD` to get the latest from wwcohen.

Eventually it will be available in regular QEMU, but probably not until the 7.0 release.

**Comment by baude** [comment]
Please try my tap.  It will be what we likely merge:
```
$ brew uninstall podman
$ brew uninstall qemu
$ brew tap baude/podman
$ brew install baude/podman/podman
```
